src/frontend/apps/drive/src/features/explorer/components/app-view/DefaultAppViewWithFilter.tsx

```diff
@@ -0,0 +1,35 @@
+import { ItemFilters } from "@/features/drivers/Driver";
+import { AppExplorer } from "@/features/explorer/components/app-view/AppExplorer";
+import { useState, useMemo } from "react";
+import { useInfiniteItems } from "@/features/explorer/hooks/useInfiniteItems";
+
+export type DefaultAppViewWithFilterProps = {
+  defaultFilters: ItemFilters;
+};
+export default function DefaultAppViewWithFilter({
```

Comment:
The name of this component is quite harsh to understand imho. 
- `WithFilter`: you can already filter with the `AppExplorer`.
- `Default`: why is it default has it is not generic, it only treats roots `items` 

Anyway, here is my proposal:

As this component is only used to display root workspaces, maybe name it `WorkspacesExplorer` and put it in `features/explorer/components/workspaces-explorer`. I think it would make more sense. WDYT?